docs/api/url_reference/logicalnot_criterion.md

```diff
@@ -0,0 +1,18 @@
+# LogicalNot Criterion
+
+The [`LogicalNot` URL Criterion](https://github.com/ezsystems/ezplatform-kernel/blob/v1.0.0/eZ/Publish/API/Repository/Values/URL/Query/Criterion/LogicalNot.php)
+matches a URL if the provided criterion does not match.
+
+It takes only one criterion in the array parameter.
+
+## Arguments
+
+- `criterion` - represents the criterion that should be excluded from the results
```

Comment:
```suggestion
- `criterion` - represents the Criterion that should exclude a URL from the results
```
not sure about this, but maybe clearer this way

Reply:
```suggestion
- `criterion` - represents the Criterion that should be negated
```